src/ripple/nodestore/ShardSizeTuning.md

```diff
@@ -0,0 +1,212 @@
+# Shard Size Tuning
+
+The purpose of this document is to compare the sizes of shards containing
+varying amounts of ledgers.
+
+## Methodology
+
+One can see visually from a block explorer that typical mainnet ledger
+consists of about 30 offer transactions issued by about 8 different accounts,
+and several transactions of other types. To simulate this situation and
+similar situations we had constructed deterministic shards of differenet sizes,
+with different number of offer transactions per ledger and different number
```

Comment:
Consider: "with varying amounts of offers per ledger and varying amounts of accounts issuing..."

Reply:
Done.